_posts/2020-01-26-nunit-console-3.11.md

```diff
@@ -7,4 +7,4 @@ categories: news update nunit
 
 Version 3.11 of the NUnit Console and Engine has now been released. This release fixes a range of minor bugs and includes a significant amount of internal restructuring work. In future, this will enable improved .NET Standard support in the engine and a .NET Core build of the console.
 
-You may download NUnit Console 3.10 from [Github](https://github.com/nunit/nunit-console/releases). See the [release notes](https://github.com/nunit/docs/wiki/Console-Release-Notes) for more information.
+You may download NUnit Console 3.11 from [Github](https://github.com/nunit/nunit-console/releases). See the [release notes](https://github.com/nunit/docs/wiki/Console-Release-Notes) for more information.
```

Comment:
(missed earlier, they capitalize it this way)

```suggestion
You may download NUnit Console 3.11 from [GitHub](https://github.com/nunit/nunit-console/releases). See the [release notes](https://github.com/nunit/docs/wiki/Console-Release-Notes) for more information.
```

Reply:
Good spot - thanks!